scan/src/contexts/AccountContext.re

```diff
@@ -26,11 +35,45 @@ let context = React.createContext(ContextHelper.default);
 let make = (~children) => {
   let (state, dispatch) = React.useReducer(reducer, None);
 
+  let sendRequest =
```

Comment:
This is bad. Do not send a function to user as a state. Let's have user dispatch action, similar to Connect.